tests/test_parquet.py

```diff
@@ -113,5 +131,76 @@ def testMultiIndexDataFrame(self):
             self.butler.get(self.datasetType, dataId={}, parameters={"columns": ["d"]})
 
 
+@unittest.skipUnless(pd is not None, "Cannot test DataFrameDelegate without pandas.")
+class DataFrameDelegateTestCase(unittest.TestCase):
+    """Tests for DataFrameDelegate, for use with InMemoryDatastore."""
+
+    def testSingleIndexDataFrame(self):
+        data = np.zeros(5, dtype=[("index", "i4"), ("a", "f8"), ("b", "f8"), ("c", "f8"), ("ddd", "f8")])
```

Comment:
If the two datasets are remaining identical to the ones in the other test case consider having a helper function to generate them.